src/main/java/ch/jalu/configme/Comment.java

```diff
@@ -22,4 +22,12 @@
      */
     String @NotNull [] value();
 
+    /**
+     * Defines if the comment should be repeated if it would be included multiple times. Relevant only for bean
```

Comment:
Think of rephrasing "if it would be included multiple times". Not sure how though.